content/ch-quantum-hardware/ac-Stark-shift.ipynb

```diff
@@ -46,10 +46,12 @@
     "# Importing standard Qiskit libraries and configuring account\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    qc_stark.add_calibration(stark_gate, (0,), stark_pulse, [amplitude, drive_freq, control_freq])
</pre><p>Nitpick; You don't need to explicitly write parameters. <code>(stark_gate, (0,), stark_pulse)</code> will work because the method automatically extract parameters attached to gate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qiskit-community/qiskit-textbook/pull/1455/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>OK. I will delete a unnecessary code.</p>